tsconfig.json

```diff
@@ -9,7 +9,8 @@
     "outDir": "dist",
     "sourceMap": true,
     "strict": true,
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "skipLibCheck": true
```

Comment:
I guess it's a good short term solution but it would be better to not have it.

How do you explain the app is currently working fine without this flag, in development or now?

Reply:
I added some missing types. Now at compilation only complains of @types/promise-events that does not exist at the moment. That's why the flag.
There is an issue in probot where they end up adding the same flag to tsconfig.json
https://github.com/probot/probot/issues/512#issuecomment-393606190

I can try to find another workaround.

Reply:
And https://github.com/probot/probot/pull/546#issuecomment-392963951

Reply:
Ok, interesting.

Reply:
I opened a pull-request at [DefinitelyTyped repos](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/40359) to create the **@types/promise-events** package with the bare minimum. But I think it can take a long time and even never be approved. 
The fastest option would be to create the ```src/@types/promise-events``` directory with the events.d.ts file inside. This is the workaround that probot ended up implementing in its source code. 
I tried this solution, it allows us to skip the flag ```"skipLibCheck": true```